SU-SOE/stanford_soe_helper
# 7.x-2.x-giving-staff

# READY FOR REVIEW

# Summary
- This adds the views and contexts to display both staff and giving spotlights

# Needed By (Date)
- Sprint end

# Criticality
- Client is eagerly awaiting this.

# Steps to Test
- Switch to this branch
` git clone git@github.com:SU-SOE/stanford_soe_helper.git`
`git checkout 7.x-2.x-giving-staff` 
` drush cc all`
- Check to see if _Stanford People Spotlight views_ or  _Stanford People Spotlight Layouts_
 need to be reverted (admin/structure/features). They didn't on need to be reverted on _Dev_.
`drush fd`
- Navigate to spotlight/staff and verify the view (the layout should look just like spotlight/faculty)
- If necessary, create some test _giving_ spotlights at `add/stanford-people-spotlight`
- Navigate to spotlight/giving and verify the view
(the layout for spotlight/giving  should look just like spotlight/faculty)
Please see images on this ticket:
https://stanfordits.atlassian.net/browse/SOE-2463

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2413
https://stanfordits.atlassian.net/browse/SOE-2463
## Related PRs

## More Information

## Folks to notify
@kbrownell 


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
